docs/common/top-bar.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, SvgIcon, Typography } from '@mui/material';
+import * as React from 'react';
+
+interface Props {
+  icon?: React.ElementType;
+  title: JSX.Element | string;
+}
+
+const TopBar: React.FC<Props> = ({ icon, title }) => {
+  return (
+    <Box
+      sx={{
+        height: '68px',
+        bgcolor: '#555555',
+        display: 'flex',
+        alignItems: 'center',
+        padding: '0px 1rem',
+        justifyContent: 'flex-start',
+      }}
+    >
+      {icon && (
+        <SvgIcon
+          component={icon}
+          viewBox="0 0 35 35"
+          sx={{ width: '34px', height: '34px' }}
+        />
+      )}
+      <Typography sx={{ color: '#FFFFFF', marginLeft: '1rem' }}>{title}</Typography>

```

Comment:
For this color and `#555555`, I'd suggest getting colors from the theme palette, get theme with `useTheme`.  If possible, we should _never_ see hex codes in our component code.  Only references to the palette.